### DISCUSSION TITLE
Include the generated SQLite database file as an asset when making releases

### DISCUSSION THREAD

**Discussion by omniuni** [PokeAPI/pokeapi#1096] [Ideas]
It looks like it can be generated from the source, but it would be really helpful to just be able to grab the SQLite file from the releases page!

**Comment by Naramsim** [comment]
Yes, I was thinking about that. Maybe even a Postgres dump. I wonder about the size though

**Comment by omniuni** [comment]
How big is the SQLite database? They're typically fairly compact, even with a lot of data in them.

**Reply by Naramsim** [reply]
Hi! Could you try build it and get the size?

**Reply by omniuni** [reply]
I'll give it a shot later today. I was hoping someone might have it sitting around to check really, but I'll put on my grown-up-developer shoes and see what I can manage. Thanks for taking the time to reply, and your efforts to maintain the project!

**Reply by Naramsim** [reply]
Thanks for your time! If you can come up with the postgres dump or SQLite db size it would be great! You find all the info for building the db on the readme or in the cicd files

**Reply by Naramsim** [reply]
Hi, i managed to make it happen: https://github.com/PokeAPI/pokeapi/releases/tag/2.9.0

You should find a DB dump, made with this command: `pg_dump -h localhost -Fc -U ash pokeapi | gzip > pokeapi.dump.zip`. Could you try and load it on your side?

**Reply by filipesilva** [reply]
Heya, I restored it succesfully onto an existing db by doing this:
- downloaded the dump from https://github.com/PokeAPI/pokeapi/releases/download/2.9.0/pokeapi.dump.zip
- unzipped it to `~/Downloads/pokeapi.dump`
- connected to the db with `psql -h my-host -U my-user`
  - replace `my-host` and `my-user` with yours, on this and other commands
- created a new db called `pokeapi` with `create database pokeapi`
- closed `psql` with ctrl+d
- restored using `pg_restore -h my-host -U my-user -d pokeapi --no-owner --no-privileges --clean --if-exists -v ~/Downloads/pokeapi.dump`
- connected with `psql -h my-host -U my-user pokeapi`
- saw tables with `\d`

First time I tried to restore I had errors like `pg_restore: error: could not execute query: ERROR:  role "ash" does not exist` so I added the flags to not set ownership or privileges.

Thanks for the great work!

**Reply by Naramsim** [reply]
Hi, thanks for the feedback ! You can also check here how I did the import for our tests: 

https://github.com/PokeAPI/pokeapi/blob/master/.github%2Fworkflows%2Fdatabase.yml#L69
